src/assets/icons/icon-checkmark.svg

```diff
@@ -0,0 +1 @@
+<svg width="16" height="12" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M5.87 11.27a1.1 1.1 0 01-1.56 0L.8 7.72C.39 7.28.4 6.6.82 6.18a1.1 1.1 0 011.55 0L5.1 8.92l8.54-8.6a1.1 1.1 0 011.55 0c.43.43.43 1.13 0 1.57l-9.31 9.38z" fill="#fff"/></svg>
```

Comment:
Could we use our default `@fortawesome` icons?

Reply:
https://github.com/sparkletown/sparkle/pull/1440/commits/c7176f5f7229863de63943d302f579e2424e3b20